students/Vincent_Aquila/session03/slicing_lab.py

```diff
@@ -0,0 +1,65 @@
+"""
+Python210A Vincent Aquila  Fall2018
+using Python 3.7.0
+assignment objective: Slicing Lab
+
+Write some functions that take a sequence as an argument, and return a copy of that sequence:
+
+1) with the first and last items exchanged.
+2) with every other item removed.
+3) with the first 4 and the last 4 items removed, and then every other item in the remaining sequence.
+4) with the elements reversed (just with slicing).
+5) with the last third, then first third, then the middle third in the new order.
+NOTE: These should work with ANY sequence – but you can use strings to test, if you like.
+
+Your functions should look like:
+
+def exchange_first_last(seq):
+    return a_new_sequence
+"""
+
+#item 1
+print("---Item 1: with the first and last items exchanged")
+def swap_first_last(list):
+    x = list[:1]
+    list[:1] = list[-1:]
+    list[-1:] = x
+    print(list)
+swap_first_last(list = ([1, 2, 3, 4, 5, 6, 7, 8, 9]))
+
+#item 2
+print("---Item 2: with every other item removed")
+def remove_every_other(list2):
+    print(list2[0:9:2])
```

Comment:
don't hard-code the 9! what if it was a different length???

and return it, don't print it.

but the slicing is correct :-)